Where_Is_My_Wife/Assets/_Main/Assets/Stage/Animations/RepawnPointTransitionAnimation.cs

```diff
@@ -0,0 +1,54 @@
+using UnityEngine;
+using WhereIsMyWife.Managers;
+using WhereIsMyWife.Controllers;
+
+namespace WhereIsMyWife.Controllers
+{
+        public class RespawPointTrasition : MonoBehaviour
```

Comment:
Rename RespawnTrigger, use Rider "Rename" command,  your classes dont match the name of the script.. again